engine/admin/volumes/bind-mounts.md

```diff
@@ -0,0 +1,367 @@
+---
+description: Using bind mounts
+title: Use bind mounts
+keywords: storage, persistence, data persistence, mounts, bind mounts
+---
+
+Bind mounts have been around since the early days of Docker. Bind mounts have
+limited functionality compared to [volumes](volumes.md). When you use a bind
+mount, a file or directory on the _host machine_ is mounted into a container.
+The file or directory is referenced by its full or relative path on the host
+machine. By contrast, when you use a volume, a new directory is created within
+Docker's storage directory on the host machine, and Docker manages that
+directory's contents.
+
+The file or directory does not need to exist on the Docker host already. It is
+created on demand if it does not yet exist. Bind mounts are very performant, but
+they rely on the host machine's filesystem having a specific directory structure
+available. If you are developing new Docker applications, consider using [named
+volumes](volumes.md) instead. You can't use Docker CLI commands to directly
+manage bind mounts.
+
+## Choosing the -v or --mount flag
```

Comment:
Our UX research into the CLI shows that the `--mount` syntax is easier to learn and use. With `--volume` even people using Docker on a daily basis had to go to the docs to check what was the order of the arguments.

So I think we could do new users a favor by deemphasizing `-v`. Also, if I'm a new user I don't know what I don't know, so at this point I don't have enough knowledge to make an informed decision between `--mount` or `--volume`, so you could give me a very opinionated way of using volumes/bind-mounts if that way would save me time in the long run.

So consider organizing this article slightly differently:
1. What are bind-mounts
2. Start a container with a bind mount
3. Mount into a non-empty directory
4. Use a read-only bind-mount
5. Configure bind propagation
...
x. Differences between `--mount` and `--volume`